proc/grouper_test.go

```diff
@@ -138,7 +138,7 @@ func TestGrouperProcJoin(t *testing.T) {
 }
 
 // TestGrouperNonDecreasing tests the disappearance of a process.  Its previous
-// contribution to the counts should not go away when that happens.
+// contribution to the counts should not go away when that happens if removeDeadGroups if false.
```

Comment:
if removeDeadGroups is false.

Reply:
The `false` here is right (if removeDeadGroups is false we don't want to remove the process). But in the next test I has true instead of false, so I fixed that.
I also renamed removeDeadGroups to removeEmptyGroups, because I think that's a better name

Reply:
Oh, I just understood the comment. Fixed :)